numpy/distutils/fcompiler/compaq.py

```diff
@@ -74,7 +74,7 @@ class CompaqVisualFCompiler(FCompiler):
     fc_exe = 'DF'
 
     if sys.platform=='win32':
-        from distutils.msvccompiler import MSVCCompiler
+        from numpy.distutils.msvccompiler import MSVCCompiler
```

Comment:
This line was due to Intel just a few days ago, that make me suspicious of this fix.